src/index.html

```diff
@@ -1,13 +1,413 @@
+              <a href="#" class="nav__list-link">
+                testimonials
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a href="#" class="nav__list-link">
+                contacts
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+  </header>
+
+  <section class="body">
+
+    <section class="description">
+      <div class="description__wrap">
+        <div class="description__title">
+          <h2 class="description__title-text">
+            Miami best <br> Real estate
+            <a href="#" class="description__title-text-link">
+              here
+            </a>
+          </h2>
+        </div>
+        <div class="description__text">
+          <h4 class="description__text-title">
+            Over 50 years experience in love.
+          </h4>
+          <p class="description__text-quote">
+            Mazel and VA Tsukkerman in his "Analysis
+            of musical works." <br> Pointillism, which
+            originated in the music of the early
+            twentieth <br> century, microforms, found
+            a distant historical. <br> <br>
+          </p>
+          <p class="description__text-quote">
+            The market leader in real estate since 2003.
+            Awards Real Estate <br> Company, 2010, 2011, 2013.
+            More than 200 satisfied customers <br> premium
+            segment. We focus only on new buildings and
+            high-level <br> doskanalno know this area better
+            than other specialists. <br> Working directly
+            with developers - so have more useful
+            information <br> about the profitable deals. <br> <br>
+          </p>
+          <p class="description__text-guarantee">
+            We will never be late! <br>
+            Guaranteed secure transaction - 100%
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="gallery">
+      <h2 class="gallery__title">
+        Gallery
+      </h2>
+      <img
+        src="images/crown.svg"
+        alt="crown image"
+        class="gallery__crown-img"
+      >
+      <div class="gallery__wrap">
+        <div class="gallery__marina">
+          <img
+            src="images/photos/1.jpg"
+            alt="house` photo"
+            width="673px"
+            height="256px"
+            class="gallery__img-1"
+          >
+          <img
+            src="images/photos/2.jpg"
+            alt="house` photo"
+            width="327px"
+            height="256px"
+            class="gallery__img-2"
+          >
+          <p class="gallery__text">
+            <span class="gallery__span">Marina Palms</span> / North Miami Beach, FL 33162
+          </p>
+        </div>
+        <div class="gallery__edition">
+          <img
+            src="images/photos/3.jpg"
+            alt="house` photo"
+            width="500px"
+            height="256px"
+            class="gallery__img-1"
+        >
```

Comment:
```suggestion
          <img
            src="images/photos/3.jpg"
            alt="house` photo"
            width="500px"
            height="256px"
            class="gallery__img-1"
          >
```
Открывающая и закрывающая скобка должны быть на одной линии

Reply:
fixed